<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>921</width>
      <height>711</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>921</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::TabPosition::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_1">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Canvas</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsViewCanvas">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>1</y>
        <width>921</width>
        <height>671</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Console</string>
     </attribute>
     <widget class="QTextEdit" name="txtConsole">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>4</y>
        <width>911</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="matplotlibWidget" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>349</y>
        <width>901</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Py File</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>4</y>
        <width>911</width>
        <height>661</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>711</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDropMode::DragOnly</enum>
    </property>
    <column>
     <property name="text">
      <string>ToolBox</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>testing box</string>
     </property>
     <item>
      <property name="text">
       <string>subtest</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>400</y>
      <width>131</width>
      <height>331</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Projects</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22cc22;
color: white;
border-radius: 8px;
padding: 4px 10px;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEngine">
    <property name="geometry">
     <rect>
      <x>969</x>
      <y>0</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22cc22;
color: white;
border-radius: 8px;
padding: 4px 10px;
font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>Choose Engine</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actiontesting_for_edit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCustom">
    <property name="title">
     <string>Custom</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuCustom"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiontesting_for_edit">
   <property name="text">
    <string>testing for edit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
